gradle.properties

```diff
@@ -12,7 +12,7 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-aws-advanced-jdbc-wrapper.version.major=2
+aws-advanced-jdbc-wrapper.version.major=3
```

Comment:
This changes the major version, it should be for the subminor line

Reply:
Fixed